src/Nethermind/Nethermind.Network.Enr/NodeRecordSigner.cs

```diff
@@ -55,7 +55,8 @@ public NodeRecord Deserialize(RlpStream rlpStream)
     {
         int startPosition = rlpStream.Position;
         int recordRlpLength = rlpStream.ReadSequenceLength();
-
+        if (recordRlpLength > 300)
```

Comment:
Just thinking shouldn't this be `RlpException` at this level and potentially packed to `NetworkingException` on different level?

Reply:
i mean, the RLP as not invalid. it is just too big to be a valid ENR, right?

Reply:
That is a good philosophical question

Reply:
Where is this called?